design/README.md

```diff
@@ -19,9 +20,16 @@ type actionType =
                               // This handles \vs 13b, \vs 13c, etc.
 ```
 
-If the marker is `\vs`, theres 2 additional booleans to determine special cases:
+If the marker is `\vs`, theres 3 additional booleans to determine special cases:
+* `vs_verse_bridge` - The `vs` marks a verse bridge (x-y) or [x-y]. This does special handling for verseNum (current verse counter)
```

Comment:
What is a verse bridge? (Maybe this is explained somewhere else)

Reply:
"Verse bridges"" are a way in Paratext to mark text that spans multiple verses.

For example, the txt to 2 Kings 9 has markers
`\vs (15-16)a` so the SFM for Paratext would mark it as
`\v 15-16 ....`

I think for some translations, it's used because the grammar for a particular language would rearrange verse orders